iree/turbine/kernel/boo/fusion/subgraph.py

```diff
@@ -0,0 +1,206 @@
+# Copyright 2025 The IREE Authors
+#
+# Licensed under the Apache License v2.0 with LLVM Exceptions.
+# See https://llvm.org/LICENSE.txt for license information.
+# SPDX-License-Identifier: Apache-2.0 WITH LLVM-exception
+
+from typing import Sequence, Set, List, Dict
```

Comment:
Nit: I think all of `set`, `list`, and `dict` are now subscriptable with `[]` and their capitalized equivalences from `typing` are no longer necessary.